scout/server/blueprints/variants/controllers.py

```diff
@@ -163,6 +165,80 @@ def parse_variant(store, institute_obj, case_obj, variant_obj, update=False):
     return variant_obj
 
 
+def variant_export_lines(store, case_obj, variants_query):
+    """Get variants info to be exported to file, one list (line) per variant"""
```

Comment:
Could you please expand the docstring?
It should at least include `Args` and `Returns`